### DISCUSSION TITLE
Black dashes for HTTPs calls

### DISCUSSION THREAD

**Discussion by CBiglemon** [jaegertracing/jaeger#3906] [Q&A] [answered]
Greetings,

We seem to have black dashes for our HTTPs calls and we aren't quite sure what they're supposed to mean. We found nothing resembling in the doc nor in any screenshoot we've had the chance to look at. Here's an exemple : 

![Screenshot_2022-03-17_at_13 59 45](https://user-images.githubusercontent.com/72421620/189599556-fcda8fb2-ba87-441a-9e47-8e19f168ce88.png)

In browser, the call seems to persist past the black dash but in a node environment it stops at it.

Any answer as to what those black dashes mean / stand for would be greatly appreciated 🙏

**Comment by yurishkuro** [comment, marked as answer]
These ticks indicate the timing of log records attached to the span.

**Reply by CBiglemon** [reply]
Thank you !

**Reply by CBiglemon** [reply]
Is it possible to know in which condition the ticks are shown?

**Reply by yurishkuro** [reply]
They are shown when span contains span logs. If you use OTEL SDK, it converts span events into span logs in Jaeger.

As far as who adds those entries to the span, it depends on the instrumentation.
